config.example/group_vars/all.yml

```diff
@@ -1,6 +1,17 @@
 # Configuration common to all hosts
 # Define per-group or per-host values in the other configuration files
 
+
+# Proxy Variables needed for ansible etc.
+#HTTP_PROXY
+#HTTPS_PROXY
+#NO_PROXY
+
+#PROXY_ENV
+    #HTTP_PROXY
+    #HTTPS_PROXY
+    #NO_PROXY
```

Comment:
Instead of using these comments as a target for the `sed` in `scripts/setup.sh`, let's just have the user duplicate the configuration directly. I.e., replace this text with:

```
# Configure these variables to use an HTTP proxy
# no_proxy should be used for internal Kubernetes addresses that should not be proxied.
# Values below show an example config.

# proxy_env:
#     http_proxy: "http://10.0.2.5:3128"
#     https_proxy: "http://10.0.2.5:3128"
#     no_proxy: "localhost,cluster.local,127.0.0.1,::1,10.0.2.10,10.0.2.20,10.0.2.30" 

```

It's a little extra work when for the user when configuring DeepOps, but it's less fragile against the user making other configuration changes or using an alternate path for their configuration directory.

Reply:
The premise of using the proxy.sh script was to minimize the places for users to make edits and therefore make mistakes. setup.sh needs drives the config file changes like the other and unfortunately not all ansible commands respect the group_var's variable. so the config file is setup for the user but they're free to make changes as necessary.